doc/rippled-example.cfg

```diff
@@ -824,6 +824,10 @@
 #                           require administrative RPC call "can_delete"
 #                           to enable online deletion of ledger records.
 #
+#       genesis_seq         The default is 32570 to match the XRP ledger
+#                           network's earliest allowed sequence. Alternate
+#                           networks may set this value. Minimum value of 1. 
```

Comment:
Isn't the genesis ledger usually ledger 0? Not that it makes much of a difference, especially in sharding, but it seems weird to start at height 1.